bitcoin/src/blockdata/weight.rs

```diff
@@ -54,8 +54,14 @@ impl Weight {
         vb.checked_mul(Self::WITNESS_SCALE_FACTOR).map(Weight::from_wu)
     }
 
-    /// Constructs `Weight` from virtual bytes in const context.
-    pub const fn from_vb_const(vb: u64) -> Weight {
+    /// Constructs `Weight` from virtual bytes returning the unwrapped value.  This function can be
+    /// deprecated in favor of from_vb when the MSRV supports unwrap() in consts context.
```

Comment:
Please drop the second phrase, it's not suitable for user-facing documentation.

Reply:
Hmm I suppose so.  This has been dropped.